config/migrations/20220615120000-public-decrees.sparql

```diff
@@ -0,0 +1,73 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?document ext:toegangsniveauVoorDocumentVersie ?toegangsniveau .
+  }
+}
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?document ext:toegangsniveauVoorDocumentVersie <http://themis.vlaanderen.be/id/concept/toegangsniveau/c3de9c70-391e-4031-a85e-4b03433d6266> .
+  }
+}
+WHERE {
+  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+    ?document a dossier:Stuk .
+    ?document dct:title ?title .
+    ?document ext:toegangsniveauVoorDocumentVersie ?toegangsniveau .
+    ?document ^besluitvorming:geagendeerdStuk/^dct:hasPart/besluitvorming:isAgendaVoor ?meeting .
+    ?meeting besluit:geplandeStart ?plannedStart .
+     FILTER ( ?plannedStart > xsd:dateTime("2016-09-08T00:00:00Z"))
+     FILTER ( regex(?title, ".*DEC.") )
+  }
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?toegangsniveau skos:prefLabel ?toegangsniveauLabel .
+    FILTER ( ?toegangsniveauLabel != "Publiek"@nl)
+  }
+}
+
+# This query could insert a new publication activity for each meeting.
+# We could also make a construct and separate this to a migration that is not in the repo.
```

Comment:
It's ok for me to check this in in the repos as a migration, but I would move it to a separate file that comes before this one. Otherwise I'm afraid it will get lost. 
Current `prov:startedAtTime` is ok. When deploying we can also temporarily tweak the `PUBLICATION_WINDOW_MILLIS` config on [themis-export-service](https://github.com/kanselarij-vlaanderen/themis-export-service#configuration) to enforce the publication to be picked up.

Reply:
right now, the time is set to 02/07/2022 12u (a saturday)
If the current ACC release goes to prod this weekend, and this is included in the next ACC release that may go live next weekend.